packages/react-liveness/package.json

```diff
@@ -55,12 +55,14 @@
     "@smithy/eventstream-serde-browser": "^2.0.4",
     "@smithy/fetch-http-handler": "^2.1.3",
     "@smithy/protocol-http": "^3.0.3",
-    "@tensorflow-models/blazeface": "0.0.7",
-    "@tensorflow/tfjs-backend-cpu": "3.11.0",
-    "@tensorflow/tfjs-backend-wasm": "3.11.0",
-    "@tensorflow/tfjs-converter": "3.11.0",
-    "@tensorflow/tfjs-core": "3.11.0",
+    "@mediapipe/face_detection": "~0.4.0",
+    "@tensorflow-models/face-detection": "1.0.2",
+    "@tensorflow/tfjs-backend-cpu": "4.11.0",
+    "@tensorflow/tfjs-backend-wasm": "4.11.0",
+    "@tensorflow/tfjs-converter": "4.11.0",
+    "@tensorflow/tfjs-core": "4.11.0",
     "@xstate/react": "^3.2.2",
+    "classnames": "2.3.1",
```

Comment:
We removed `classnames` as a dep and added a util to replace it in the UI package